### DISCUSSION TITLE
Cannot list alerts with importance sort order

### DISCUSSION THREAD

**Discussion by yatif** [grafana/grafana#53513] [Dashboards, Panels & Visualizations]
Hello there,

We decided to enable new Grafana Alerts, and a lot of things changed, alert list plugin(panel) included. With legacy alerting, we could sort alerts according to their importances (like firing>pending>inactive) but for new grafana alert list can’t do this. It also has the “importance” option, but I am not sure what it is for.
Am I wrong? Should not “importance sort order” sort alerts according to their states? If I am wrong, can you add this useful feature to grafana.

Thank you

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
